BoldGrid/w3-total-cache
# Add help tabs and content

Adds tutorial content for main caching types

# Testing

1. Go to each of the main caching types.
2. Click on the tutorials tab
3. Review the content there 
4. Click back to general settings or premium services
5. Make sure the tabs are working correctly with hiding and showing the correct information.
